### DISCUSSION TITLE
Animation in battey saver mode

### DISCUSSION THREAD

**Discussion by idimas13** [zepp-health/zeppos-forum#117] [Q&A]
In the "battery saver mode" the watch shows the middle frame of the animation of the watchface. This is inconvenient way for animations. There should be the last frame shown.

**Comment by MyoungXUE** [comment]
Hi, you can first try the `default_frame_index` property to specify the index of the frame, this property only works on Zepp OS 2.0 devices.

We will improve the documentation for this part as soon as possible.

**Reply by idimas13** [reply]
And again, do we have this "default_frame_index" setting in the Watchface maker?
No, we have to dig into the code. And before that, we have to dig into zab, zpk etc...
